distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -4,9 +4,9 @@
 # update openHAB one day.
 
 
-####################################################################################### 
+#######################################################################################
```

Comment:
Wow, did not mean to change this file so much, my atom editor must have done some whitespace editing, I'll revert this and try again!